common/app/experiments/LookedAtExperiments.scala

```diff
@@ -0,0 +1,20 @@
+package experiments
+
+import play.api.libs.typedmap.TypedKey
+import play.api.mvc.RequestHeader
+
+import scala.collection.mutable
+
+object LookedAtExperiments {
+
+  private val attrKey: TypedKey[mutable.Set[Experiment]] = TypedKey[mutable.Set[Experiment]]("lookedAtExperiments")
```

Comment:
Might be dangerous using a mutable.Set for multi threaded behaviour. Could use a Java ConcurrentHashMap and convert that to a Scala Set when reading.